tools/mmp/driver.cs

```diff
@@ -1374,14 +1387,22 @@ static void LipoLibrary (string name, string dest)
 			if (existingArchs.Count () < 2)
 				return;
 
-			var arch = App.Abi.AsString ();
 			// macOS frameworks often uses symlinks and we do not
 			// want to replace the symlink with the thin binary
 			// while leaving the fat binary inside the framework
 			dest = GetRealPath (dest);
-			RunLipo (App, new [] { dest, "-thin", arch, "-output", dest });
-			if (name != "MonoPosixHelper" && name != "libmono-native-unified" && name != "libmono-native-compat")
-				ErrorHelper.Warning (2108, Errors.MM2108, name, arch);
+
+			var prefix = new [] { dest };
+			var suffix = new [] { "-output", dest };
+			List <string> archArgs = new List <string> ();
+			foreach (var abi in App.Abis)
+			{
+				archArgs.Add ("-extract_family");
+				archArgs.Add (abi.ToString ().ToLower ());
```

Comment:
use `.ToLowerInvariant ()` to avoid culture sensitive code paths

Reply:
Fixed at 2067dbb0be4cf24dcf4f54d9b4d8e9ef1bafa5bc